### PR TITLE
added question 1-10

### PR DESCRIPTION
**Pull request by mrnobody3**

### CODE REVIEW

File: questions/front-end/basic.md [hunk 1]
```diff
@@ -18,21 +15,191 @@ DOM є деревоподібною структурою (дерево доку
 3. Взаємодією з DOM: async може вплинути на DOM під час виконання, тоді як defer не впливає на DOM, оскільки виконується лише після завантаження всього DOM.
 
 ### 3. Яка різниця між властивостями HTML-елементів innerHTML і innerText?
```
Comment (SerhiienkoRuslan) [inline questions/front-end/basic.md]: use this ##

File: questions/front-end/basic.md [hunk 2]
```diff
@@ -18,21 +15,191 @@ DOM є деревоподібною структурою (дерево доку
...
+І `innerHTML`, і `innerText` є властивостями елементів HTML. Ми можемо змінити вміст елемента HTML за допомогою цих властивостей.
+
+Ми можемо призначити рядок HTML властивості `innerHTML`, яка відображається як звичайний HTML. Перевірте наведений нижче приклад.
+
+    const titleEl = document.getElementById("title");
```
Comment (SerhiienkoRuslan) [inline questions/front-end/basic.md]: cover code in ```jsx```

File: questions/front-end/basic.md (hunk 1 above)
Comment (mrnobody3) [reply questions/front-end/basic.md]: done

File: questions/front-end/basic.md (hunk 2 above)
Comment (mrnobody3) [reply questions/front-end/basic.md]: done

File: questions/front-end/basic.md [hunk 3]
```diff
@@ -17,56 +14,248 @@ DOM є деревоподібною структурою (дерево доку
...
 
-### 4. Опишіть процес спливання (bubbling) подій у DOM.
+Ми можемо призначити рядок HTML властивості `innerHTML`, яка відображається як звичайний HTML. Перевірте наведений нижче приклад.
 
-### 5. Як зупинити спливання (bubbling) події?
+```jsx
+const titleEl = document.getElementById("title");
+titleEl.innerHTML = '<span style="color:orange;">techukraine.net</span>';
+```
+Додайте один елемент із заголовком id до свого HTML і додайте наведений вище сценарій до файлу JavaScript. Запустіть код і подивіться результат. Ви будете techukraine.net в оранжевому кольорі. І якщо ви перевірите елемент, він буде всередині тегу span. Отже, `innerHTML` візьме рядок HTML і відтворить його як звичайний HTML.
```
Comment (SerhiienkoRuslan) [inline questions/front-end/basic.md:26]: Ви будете techukraine.net в оранжевому кольорі. 

what is this?

**Review by SerhiienkoRuslan — APPROVED** [review]

**Pull request merged**